handlers/error.go

```diff
@@ -61,7 +61,9 @@ const (
 	ErrFetchKubernetesCode      = "2051"
 	ErrPanicRecoveryCode        = "2052"
 	ErrBlankNameCode            = "2053"
+	ErrSaveSessionCode          = "2054"
 	ErrInvalidLTURLCode         = "2053"
+
```

Comment:
I think we can remove all the spaces that are not required let's see if this lint passes then.

Reply:
Sure Let's try this too

Reply:
You see my commits are signed off then also it's not showing. strange!

Reply:
> You see my commits are signed off then also it's not showing. strange!

Gotcha !! the files which were not required was deleted manually from Github, that's why it's rejecting DCO